spec/lib/geocode_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe Geocode do
+  it 'normalises the postcode' do
+    expect(Geocoder).to receive(:coordinates).with('RG11GG, United Kingdom')
+
+    Geocode.call('rg1 1gg')
+  end
+
+  it 'returns the lat/long pair' do
+    VCR.use_cassette(:rg11gg) do
+      latitude, longitude = Geocode.call('RG11GG')
+
+      expect(latitude).to  be
+      expect(longitude).to be
```

Comment:
If you're going to go down that route I could change the implementation of this method to return an entirely canned response that matches the expectations set by the test.

Try pulling the project down and making this change and seeing which tests fail.